aequilibrae/project/about.py

```diff
@@ -0,0 +1,82 @@
+import sqlite3
+import string
+from warnings import warn
+from aequilibrae.project.project_creation import create_about_table
+
+
+class About:
+    """Provides an interface for querying and editing the **about** table of an AequilibraE project"""
+
+    def __init__(self, conn: sqlite3.Connection):
+        self.__characteristics = []
+        self.__conn = conn
+        if self.__has_about():
+            self.__load()
+
+    def create(self):
+        """Creates the 'about' table for project files that did not previously contain it"""
+
+        if not self.__has_about():
+            create_about_table(self.__conn)
+            self.__load()
+        else:
+            warn('About table already exists', Warning)
+
+    def add_info_field(self, info_field: str) -> None:
+        """Adds new information field to the model
+
+            Args:
+                *info_field* (:obj:`str`): Name of the desired information field to be added.  Has to be a valid
+                Python VARIABLE name (i.e. letter as first character, no spaces and no special characters)
+
+            ::
+
+                p = Project()
+                p.open('my/project/folder')
+                p.about.add_info_field('my_super_relevant_field')
+                p.about.my_super_relevant_field = 'super relevant information'
+                p.about.write_back()
+        """
+        allowed = string.ascii_lowercase + '_'
+        passed = [x for x in info_field if x not in allowed]
+
+        if not passed:
```

Comment:
This looks the wrong way around... if not passed insert it?
Also, is passed actually a "fixed" version of info_field? passed implies a boolean to me

Reply:
The problem here is that the variable for this list has the wrong name.  It should be called has_forbidden_characters. Should be clear now